library/Notifications/Common/Icons.php

```diff
@@ -41,4 +41,32 @@ private function __construct()
     public const RULE_MATCHED = 'filter';
 
     public const UNDEFINED = 'notdef';
+
+    public const ACKNOWLEDGED = 'check';
+
+    public const UNACKNOWLEDGED = 'xmark';
+
+    public const DOWNTIME = 'plug';
+
+    public const FLAPPING = 'bolt';
+
+    public const INCIDENT_AGE = 'hourglass-end'; //TODO: icon? and for custom?
+
+    public const SEVERITY_OK = 'heart';
+
+    public const SEVERITY_CRIT = 'circle-exclamation';
+
+    public const SEVERITY_WARN = 'exclamation-triangle';
+
+    public const SEVERITY_ERR = 'circle-xmark';
+
+    public const SEVERITY_DEBUG = 'bug-slash';
+
+    public const SEVERITY_INFO = 'info';
+
+    public const SEVERITY_ALERT = 'bell';
+
+    public const SEVERITY_EMERG = 'tower-broadcast';
```

Comment:
I'd prefer `bullhorn`. It matches the general style of severity icons better. (Solid and bulky)